model/validate.php

```diff
@@ -220,11 +217,14 @@ function validateImageUpload($file, $newName)
                 $isValid = false;
             }
 
-            //checking for duplicate
-            if (file_exists($dirName . $newName)) {
-                $f3->set("errors['duplicatedImage']", "Sorry! This image is already exist choose another one");
-                $isValid = false;
-            } else {
+            //since we have the used the tutor's id and name as image file name to store in our db
+            // we know file name is going to be unique so no need check for duplicates
+//            if (file_exists($dirName . $newName)) {
+//                echo $dirName.$newName;
+//                $f3->set("errors['duplicatedImage']", "Sorry! This image is already exist choose another one");
+//                $isValid = false;
+//            }
+        else {
```

Comment:
Something looks very odd about the indentation levels in this function.

Reply:
Should be fixed